render/src/umi/mod.rs

```diff
@@ -80,31 +97,45 @@ impl Umi {
         sheet.set_value(0, 0, "position");
         sheet.set_value(0, 1, "id");
         sheet.set_value(0, 2, "kind");
-        sheet.set_value(0, 3, "depth");
+        sheet.set_value(0, 3, "properties");
+        sheet.set_value(0, 4, "depth");
         sheet.set_value(
             0,
-            4,
+            5,
             String::from("content-") + self.lang.to_string().as_str(),
         );
         sheet.set_value(
             0,
-            5,
+            6,
             String::from("attributes-") + self.lang.to_string().as_str(),
         );
 
-        // Row 1: Config
+        let mut hashmap: HashMap<String, String> = HashMap::new();
+        hashmap.insert(
+            String::from("input_path"),
+            self.config.input.to_str().unwrap_or_default().to_string(),
+        );
+        let properties = serde_json::to_string(&hashmap).unwrap_or(String::from("{}"));
```

Comment:
```suggestion
        let properties = serde_json::to_string(&hashmap).unwrap_or(String::from(""));
```

to not get `{}` if no properties are set for a row entry.